iohk-monitoring/src/Cardano/BM/Backend/ExternalAbstraction.lhs

```diff
@@ -92,4 +93,14 @@ instance Pipe UnixNamedPipe where
     getLine (P h) = BS.hGetLine h
     write (P h) bs = BSC.hPutStrLn h $! bs
 
+#else
```

Comment:
`Pipe` is an instance?
And why is the instance in `UnixNamedPipe` filled with error messages for Windows?

Reply:
actually not needed on Windows. thanks.